packages/v4/src/content/design-guidelines/components/search-input/search-input.md

```diff
@@ -32,9 +32,9 @@ Another way to use search input is as a find function. This would mean that the
 
 3. **Optional navigation:** The navigation feature allows the user to navigate one-by-one through the matched results. It will correspond to the count that is within the badge.
 
-4. **Match highlight:** All matches will be highlighted but we can not guarantee it will contain the gray color, depending on the use case.
+4. **Match highlight:** All matches will be highlighted using `GOLD-50`from the [PatternFly color palette](/guidelines/colors#color-palette).
 
-5. **Current highlight:** The active match will contain a different colored highlight. We can not guarantee it will contain the blue highlight, depending on this use case. In this case, since the user is on 2/5, “Node” in “Node 2” is highlighted in blue.
+5. **Current highlight:** The active match will be highlighted with `GOLD-100` from the [PatternFly color palette](/guidelines/colors#color-palette). In this case, since the user is on 2/5, “Node” in “Node 2” is highlighted.
```

Comment:
would maybe add the word "match" before "2/5" so that it reads "since the user is on match 2/5, "Node"..." just to explain a bit more what 2/5? 

Also same comment about gold-100 here!